cmd/apiserver-watcher/run.go

```diff
@@ -175,6 +176,18 @@ func (h *handler) onSuccess() error {
 	if err := removeVipStateFile(h.vip, "down"); err != nil {
 		return err
 	}
+	ip := net.ParseIP(h.vip)
+	var nlFamily netlink.InetFamily
+	nlFamily = netlink.FAMILY_V4
+	if ip.To4() == nil && ip.To16() != nil {
+		nlFamily = netlink.FAMILY_V6
+	}
+	// bz1930457 delete stale conntrack entries to the apiserver
+	filter := &netlink.ConntrackFilter{}
+	filter.AddIP(netlink.ConntrackNatSrcIP, ip)
+	if _, err := netlink.ConntrackDeleteFilter(netlink.ConntrackTable, nlFamily, filter); err != nil {
+		glog.V(4).Infof("Error deleting conntrack entries for %s", h.vip)
```

Comment:
q: we only need to log this?

Reply:
yeah, if there are no conntrack stale entries there is no problem, but this can give some info trying to debug problems

Reply:
ups, I forget to add the error message